script/training_db_sync.rb

```diff
@@ -138,13 +156,11 @@ class CmdRunner
     require "open3"
 
     def self.run(command)
-      begin
-        stdout, _stderr, _status = Open3.capture3(command)
-      rescue => error
-        raise "'#{command}' failed (#{error})"
-      end
+      _stdout, stderr, _status = Open3.capture3(command)
 
-      Kernel.puts stdout.chomp
+      if stderr
```

Comment:
I'm not sure how `capture3` works - does it rescue errors and store them in `stderr`, or do we need to keep that `rescue` in here too (rather than removing it) to be safe?

Reply:
So, as far as I can tell from docs, the debug process I went through, and some manual testing, the error is always stored in stderr - I don't think this was ever throwing the the rescue block in the first place.